features/fixtures/formatters/passed-rule.message.json.ts

```diff
@@ -0,0 +1,142 @@
+module.exports = [
+  {
+    source: {
+      uri: 'features/a.feature',
+      data: 'Feature: a feature\n  Rule: a rule\n    Example: an example\n      Given a step',
+      media: {
+        encoding: 'UTF8',
+        contentType: 'text/x.cucumber.gherkin+plain',
+      },
+    },
+  },
+  {
+    gherkinDocument: {
+      uri: 'features/a.feature',
+      feature: {
+        location: {
+          line: 1,
+          column: 1,
+        },
+        language: 'en',
+        keyword: 'Feature',
+        name: 'a feature',
+        children: [
+          {
+            rule: {
+              location: {
+                line: 2,
+                column: 3
+              },
+              keyword: 'Rule',
+              name: 'a rule',
+              children: [
+                {
+                  scenario: {
+                    location: {
+                      line: 3,
+                      column: 5,
+                    },
+                    keyword: 'Example',
+                    name: 'an example',
+                    steps: [
+                      {
+                        location: {
+                          line: 4,
+                          column: 7,
+                        },
+                        keyword: 'Given ',
+                        text: 'a step',
+                        id: '1',
+                      },
+                    ],
+                    id: '2',
+                  },
+                },
+              ],
+            },
+          },
+        ],
+      },
+    },
+  },
+  {
+    pickle: {
+      id: '4',
+      uri: 'features/a.feature',
+      name: 'an example',
+      language: 'en',
```

Comment:
Should the pickle name here be some kind of concatenation of the rule name and example name, or is the example name sufficient?